src/main/webapp/src/app/recommendation-item/recommendation-item.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RecommendationItemComponent } from './recommendation-item.component';
+import { HiddenGem } from '../hidden-gem';
+
+describe('RecommendationItemComponent', () => {
+  let component: RecommendationItemComponent;
+  let fixture: ComponentFixture<RecommendationItemComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ RecommendationItemComponent ]
+    })
+    .compileComponents();
```

Comment:
indent with tab because it is part of the lines above

Reply:
I think this is the default indention from when you create the components. I checked our other spec files and it has the same indentation?